contentgrid-spring-boot-autoconfigure/build.gradle

```
@@ -28,6 +28,7 @@ dependencies {
 
     implementation platform(project(':contentgrid-spring-boot-platform'))
     implementation 'org.springframework.boot:spring-boot-autoconfigure'
+    implementation 'org.springframework.boot:spring-boot-actuator-autoconfigure'
```

```suggestion
    compileOnly 'org.springframework.boot:spring-boot-actuator-autoconfigure'
```

We don't want to pull in actuators autoconfiguration when the user does not have actuators on the classpath.